packages/valory/services/apy_estimation/service.yaml

```diff
@@ -0,0 +1,24 @@
+name: apy_estimation
+author: valory
+version: 0.1.0
+description: A set of agents reaching consensus on APY estimates
+aea_version: '>=1.0.0, <2.0.0'
+license: Apache-2.0
+fingerprint:
+  README.md: bafybeibhbkelnxxvsln677imq65vgbglmlhyxtax4iqtzempjiwcoef3gq
+fingerprint_ignore_patterns: []
+agent: valory/apy_estimation:0.1.0:bafybeiekhoukht6nk6bbptxkkdpqnwv2x2deuljhgou4m3v2wdrvyjy6wy
+number_of_agents: 4
+---
+public_id: valory/apy_estimation_abci:0.1.0
+type: skill
+models:
+  params:
+    args:
+      pair_ids:
+        uniswap_subgraph:
+        - '0xae461ca67b15dc8dc81ce7615e0320da1a9ab8d5'
```

Comment:
Surely it needs double nesting here too? @dvilelaf

Reply:
I've just added it and also in another place but it seems it breaks tests.